ops/dev/main.tf

```diff
@@ -14,6 +15,13 @@ resource "azurerm_subnet" "vms" {
   address_prefixes     = ["10.1.252.0/24"]
 }
 
+resource "azurerm_subnet" "load_balancers" {
+  name                 = "${var.env}-load-balancers"
```

Comment:
I think we're trying to keep the naming consistent between the hub, automation and simple report applications.

So prime-simple-report-dev-lb

Reply:
sounds good